src/content/docs/opentelemetry/get-started/traceloop-llm-observability/traceloop-llm-observability-intro.mdx

```diff
@@ -0,0 +1,120 @@
+
+import llmTraceloopDetails from 'images/llm-traceloop-details_screenshot.webp'
+
+While OpenTelemetry offers a powerful standard for collecting general application data (traces, metrics, logs), it lacks the ability to capture key performance indicators (KPIs) specific to AI models. This includes crucial metrics like model name, version, prompt and completion tokens, and temperature parameters. These details are essential for effectively monitoring and troubleshooting AI model performance.
+
+OpenLLMetry emerges as a solution, specifically designed to address this gap in AI model observability. Built on top of the OpenTelemetry framework, OpenLLMetry provides seamless integration and extends its capabilities. It offers support for popular AI frameworks like OpenAI, HuggingFace, Pinecone, and LangChain.
+
+<DNT>**Key Benefits of OpenLLMetry:**</DNT>
+* <DNT>**Standardized Collection of AI Model KPIs:**</DNT> OpenLLMetry ensures consistent capture of essential model performance metrics, enabling comprehensive observability across diverse frameworks.
+* <DNT>**Deeper Insights into LLM Applications:**</DNT> With its open-source SDK, OpenLLMetry empowers you to gain thorough understanding of your Large Language Model (LLM) applications.
+This page describes common set up steps for OpenTelemetry based APM monitoring
+with New Relic.
+
+Overall, OpenLLMetry empowers developers to leverage the strengths of OpenTelemetry while gaining the additional functionalities required for effective AI model monitoring and performance optimization.
+
+<Steps>
+  <Step>
+## Before you start [#prereqs]
+
+* [Sign up](https://newrelic.com/signup) for a New Relic account.
+* Get the [license key](https://one.newrelic.com/launcher/api-keys-ui.launcher)
+  for the New Relic account to which you want to report data.
+  </Step>
+
+  <Step>
+## Instrument your LLM Model with OpenLLMetry [#instrument]
+
+OpenLLMetry is a vendor-neutral way to collect observability data from applications. 
+Since New Relic natively supports OpenTelemetry, you just need to route the traces to New Relic’s endpoint and set the API key:
+
+```env
+TRACELOOP_BASE_URL = https://otlp.nr-data.net:443
+TRACELOOP_HEADERS = "api-key=<YOUR_NEWRELIC_LICENSE_KEY>"
+```
+
+* The OpenTelemetry Protocol (OTLP) exporter to send data to the
+   [New Relic OTLP endpoint](/docs/opentelemetry/best-practices/opentelemetry-otlp).
+
+<DNT>**Example: OpenAI LLM Model with LangChain**</DNT>
+  ```python
+  from traceloop.sdk import Traceloop
+  import os
+  import time
+  from langchain_openai import ChatOpenAI
+  from traceloop.sdk.decorators import workflow, task
+
+  os.environ['OPENAI_API_KEY'] = 'OPENAI_API_KEY'
+  os.environ['TRACELOOP_BASE_URL'] = 'https://otlp.nr-data.net:443'
+  os.environ['TRACELOOP_HEADERS'] = 'api-key=YOUR_NEWRELIC_LICENSE_KEY'
+
+  Traceloop.init(app_name="llm-test", disable_batch=True, api_key="TRACELOOP_API_KEY")
+
+  @task(name="add_prompt_context")
```

Comment:
annotations are not needed if you're using LangChain since they're added automatically

Reply:
removed.